js/build.js

```diff
@@ -9,181 +14,153 @@ Fliplet.Widget.instance({
+
+                          isConditionIncluded(currentArray, condition);
+                        } else {
+                          expression = user[userKey]
+                            .split(',')
+                            .map(el => el.trim())
+                            .includes(decodeHTMLEntities(condition.user_value));
+                          setResult(evaluate(condition, expression));
+                        }
+                      } else {
+                        // other type but array or string
+                        expression = '"' + user[userKey] + '".indexOf("' + condition.user_value + '") > -1';
+                        setResult(evaluate(condition, expression));
                       }
-                    } else if (keyType === 'array') {
-                      ifArrayIncludes(user[conditions[i]['user_key']], conditions[i]);
                     }
-                  }
-                } else {
-                  if (environment) {
-                    Fliplet.UI.Toast('User doesn\'t contain key: ' + conditions[i]['user_key']);
+                  } else if (isPreview) {
+                    Fliplet.UI.Toast(`User doesn't contain key: ${userKey}`);
                   }
                 }
               }
-            }
 
-            if (result) {
-              $(helper.el).removeClass('hidden');
-            }
-          } else {
-            if (environment) {
-              Fliplet.UI.Toast('User is not logged in');
+              if (result) {
+                $(helper.el).removeClass('hidden');
+                await Fliplet.Widget.initializeChildren(helper.$el, helper);
+              }
+            } else if (isPreview) {
+              Fliplet.UI.Toast(userNotLoggedMessage);
             }
-          }
-        } else {
-          if (environment) {
-            Fliplet.UI.Toast('User is not logged in');
-          }
-        }
-        return Promise.resolve(true);
-      });
-    },
-    change: function (arrayOfFields) {
-      _.forEach(arrayOfFields, function (field) {
-        updatePanelTitleName(field);
-      });
-
-      function updatePanelTitleName(field) {
-        let visibility = field[1].value;
-        let key = field[2].value;
-        let condition = field[3].value || '';
-        let value = field[4].value;
-
-        if (visibility && key && condition && value) {
-          visibility = visibility.charAt(0).toUpperCase() + visibility.slice(1);
-          condition = condition.charAt(0).toLowerCase() + condition.slice(1);
-          if (condition.indexOf('equal') > -1) {
-            condition += ' to';
+          } else if (isPreview) {
+            Fliplet.UI.Toast(userNotLoggedMessage);
           }
 
-          let titleName = `${visibility} if "${key}" ${condition} "${value}"`;
-          field[0].value = titleName;
-        } else {
-          field[0].value = 'Incomplete condition';
-        }
-      }
+          return Promise.resolve(true);
```

Comment:
`Promise.resolve()` not need in a chain

```suggestion
          return true;
```